lib/portfinder.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * portfinder.js typescript definitions.
+ *
+ * (C) 2011, Charlie Robbins
+ */
+
+declare type PortfinderHandler = (err: Error, port: number) => void;
+
+/**
+ * The lowest port to begin any port search from.
+ */
+export declare let basePort: number;
+
+/**
+ * Responds with a unbound port on the current machine.
+ */
+export declare function getPort(callback: PortfinderHandler): void;
```

Comment:
awesome thank you

yeah the port and host are public, README needs a lil update

(technically u can pass a server option too, that is NOT public :) )

again thanks for the commit